extensions/wikia/Discussions/Discussions.setup.php

```diff
@@ -13,14 +13,18 @@
 	'url' => 'https://github.com/Wikia/app/tree/dev/extensions/wikia/SpecialDiscussions',
 ];
 
-// classes
-$wgAutoloadClasses['SpecialDiscussionsController'] = $dir . 'SpecialDiscussionsController.class.php';
+// load classes
+$wgAutoloadClasses['SpecialDiscussionsController'] = $dir . 'controllers/SpecialDiscussionsController.class.php';
+$wgAutoloadClasses['EnableDiscussionsController'] = $dir . 'controllers/EnableDiscussionsController.class.php';
+$wgAutoloadClasses['DiscussionsActivator'] = $dir . 'utils/DiscussionsActivator.class.php';
+$wgAutoloadClasses['DiscussionsVarToggler'] = $dir . 'utils/DiscussionsVarToggler.class.php';
```

Comment:
I think it's fine. Perhaps they could also reside in the extension's main folder?